roles/deploy/tasks/main.yml

```diff
@@ -99,37 +99,8 @@
   set_fact:
     timestamp: "{{ ansible_date_time.date }}-{{ ansible_date_time.hour }}{{ ansible_date_time.minute }}{{ ansible_date_time.second }}"
 
-
-#--------------------------
-# Create a rollback release
-
-- name: create a database rollback version
-  command: bash -c "set -o pipefail && pg_dump -h '{{ db_host }}' -U '{{ db_user }}' '{{ db }}' | gzip > '{{ rollback_sql_path }}'"
-  args:
-    creates: "{{ rollback_sql_path }}"
-
-- name: create a database backup version
-  command: cp {{ rollback_sql_path }} {{ releases_path }}/{{ timestamp }}.sql.gz
-  tags: skip_ansible_lint
-
-- name: create a repo rollback version
-  command: mv {{ current_path }} {{ rollback_path }} removes={{ current_path }}
-  tags: skip_ansible_lint
-
-- name: create a repo backup version
-  command: cp -r {{ rollback_path }} {{ releases_path }}/{{ timestamp }} removes={{ rollback_path }}
-  tags: skip_ansible_lint
-
-#-------------------------
-# Move new code into place
-
-- name: move build to current
-  command: mv {{ build_path }} {{ current_path }} removes={{ build_path }}
-  become: yes
-  become_user: "{{ unicorn_user }}"
-
 #--------------------
-# Update the database
+# Create database if it doesn't exist
 
 - name: check database status
```

Comment:
The diff isn't very clear here, but basically the tasks "check database status", "create database if it doesn't exist" and "load schema if it isn't loaded" have been shifted upwards, so they run before the task "create a database rollback version".